content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -36,148 +28,65 @@ We recommend using the pack defaults. Default settings provide best practices fo
+| `certRenewal` | Self-signed certificate renewal in hours | 720h (30d) |
+| `enableInsecureLogin` | RBAC ensures secure login. | `false` |
+| `serviceType` | The service type for the dashboard. Supported values are `ClusterIP`, `LoadBalancer`, and `NodePort`. | `ClusterIP` |
+| `skipLogin` | Enables or disables the skip login option on the Spectro Kubernetes Dashboard. | `false` |
+| `enableInsecureLogin` | Enables non-Secure Sockets Layer (SSL) login. Dashboard login is always restricted to HTTP(S) + localhost or HTTPS and external domain. | `false` |
+| `ingress.enabled` | Ingress configuration to access the `ClusterIP`, `LoadBalancer`, or `NodePort`. | `false` |
 
 ## Usage
 
-To use the Spectro Kubernetes Dashboard pack, you have to add it to your cluster profile. Spectro Kubernetes Dashboard supports the following clouds:
+To use the Spectro Kubernetes Dashboard pack, you have to add it to your cluster profile. Spectro Kubernetes Dashboard supports several public cloud and data center cloud environments. To learn more, refer to [Clusters](/clusters).
 
-- Amazon Web Services (AWS)
-- Amazon Elastic Kubernetes Service (EKS)
-- Canonical Metal as a Service (MAAS) 
-- Google Cloud Platform (GCP)
-- Microsoft Azure
-- OpenStack
-- VMware vSphere
+Use the following information to find the Kubernetes Dashboard pack.
+- **Pack Type**: Monitoring
+- **Registry**: Public Repo
+- **Pack Name**: Spectro Kubernetes Dashboard
+- **Pack Version**: 2.7.0 or higher
 
+Spectro Kubernetes Dashboard has the following Access options.
 
-Spectro Kubernetes Dashboard has the following default settings for **Access** and **Identity Provider**. Changing these default settings may require some additional configuration.
 <br /> 
 
-#### Access
-
-The default setting is **Private**. When **Access** is set to **Private**, the latest version of the Spectro Proxy pack is included with Spectro Kubernetes Dashboard when you create your cluster profile.
-
-If you change the setting to **Public** and your cluster is in a public cloud, there is no additional configuration.  
-
-However, if you change **Access** to **Public** and your cluster is in a private cloud, you have to manually add and configure the Spectro Proxy pack to your cluster profile. For more information, refer to the [Spectro Proxy](/integrations/frp) guide.
-
-<br /> 
-
-#### Identity Provider 
-
-All IDP options below require you to map a set of users or groups to a Kubernetes RBAC role. There are two options you can use to get started with the Kubernetes Dashboard and an IDP.
+- **Proxied**: This option provides private access to your cluster when the cluster is in a private cloud. This option requires the Spectro Proxy pack, which is integrated with the Spectro Kubernetes Dashboard. The latest version of the Spectro Proxy pack is automatically installed when you create the cluster. To learn more, check out the [Spectro Proxy](/integrations/frp) guide.
```

Comment:
```suggestion
- **Proxied**:  This option is useful for access to the Kubernetes Dashboard from anywhere. It will leverage the Spectro Proxy pack for this, which gets automatically installed if not present. The Service resource for the Kubernetes Dashboard will be configured as ClusterIP and is only accessible through the proxy. To learn more, check out the [Spectro Proxy](/integrations/frp) guide.
```

Reply:
I committed this, but it doesn't seem to indicate that.

Reply:
I added a slightly worded version of this to the doc. Thank you @kreeuwijk